local.css

```diff
@@ -176,12 +176,15 @@ a.termref:active {
 	text-decoration:none;
 	border-bottom: 1px dotted #FC0; 
 	}
+	
+.codepoint bdi {
+    font-size: 120%;
+}
 
 .uname {
 	text-transform: uppercase;
-	font-size: 85%;
-	letter-spacing:0.03em;
-	}
+	font-variant: all-small-caps;
```

Comment:
Reasons i didn't use font-variant:
1. i'm not clear what that improves
2. there is no mix of upper and lower case here
3. i don't want the browser to synthesise glyphs
4. it looks too small
5. SC font glyphs are not matched to caseless characters around them

In addition, the letter-spacing value was chosen with care, and following guidance from R Bringshurst, to make the uppercase letters more readable and compact, so please keep that, too.

Reply:
Actually, the one change i would suggest is to remove the uppercase setting.  It was only ever there as bulletproofing, but actually it would be better to be able to see when someone has typed a Unicode name or code point value in lowercase, because then it's more likely to be fixed before someone tries to copy paste it somewhere.

Reply:
Thanks @r12a 

I had kept the uppercasing because I assumed it was there for bulletproofing. Should I remove it and keep the remainder? Perhaps suggest what the complete style should be.

Side note that you might have missed: I removed the `:lang` from the `.codepoint bdi` because I couldn't see why we wouldn't style non-English-language glyphs larger. Users could still style specific language glyphs to use a `font-family` or resize to their needs by using their own `:lang`. Thoughts?

Reply:
What i'm suggesting is to make it just:
````
.uname {
    font-size: 85%;
    letter-spacing:0.03em;
    }
````
I'm flexible about the font-size value. Could be 80%.

Reply:
For the uname styling, we used to have margin-inline:2px. I think that was there to clear the [..] bracketing. i'm not sure whether we need to reinstate some margin-inline-start spacing, but probably not, looking at the results in my own docs - especially if we use images for the character itself, since the images tend to have some padding built in.

Reply:
Thought: should we color `.uname`? When I imagine removing the `[...]` bracketing I can see how the styled text might just disappear visually into the surrounding text. If we add some color that might be helpful. 

![image](https://user-images.githubusercontent.com/69082/229187243-e190c1bf-00c3-473c-b566-7903dc35b29b.png)

![image](https://user-images.githubusercontent.com/69082/229187536-b4f8ae07-eb43-422f-b839-9bb11687b4de.png)

Reply:
As you probably know, .unames are coloured in my docs, but they are the same colour as links.  I can't decide whether that's ok, because in my docs clicking on them opens up popup windows.  But for W3C docs i suppose we need to choose a colour that doesn't make it look like a link - or anything else. (I think .kw produces a blue colour.)

Reply:
Yep. Maybe avoid blue in favor of a non-link-like color. Above I used a random blue. Who could provide guidance here? fantasai?

Reply:
FWIW, UI Events uses a green colour for code points: https://w3c.github.io/uievents-code/#style-conventions

Reply:
Thanks @xfq. Green might be good, although I wouldn't use background coloring. Orange has the problem of being low contrast. Maybe the solution is to put it into the tr-design PR and just ask for color guidance.

Reply:
fwiw, i'd favour a colour that (providing it has enough contrast) doesn't jump out at you, like the blue above and the uievents styling.  I'd prefer the actual character to be the main thing that jumps out, with the number and name as backup, rather than the main event.